packages/web-app-files/src/components/SideBar/Shares/Links/NameAndCopy.vue

```diff
@@ -53,6 +60,10 @@ export default defineComponent({
       isSupported: isClipboardCopySupported
     } = useClipboard({ legacy: true, copiedDuring: 550 })
 
+    const linkCreationDate = computed(() => {
+      return DateTime.fromISO(props.linkShare.createdDateTime).toLocaleString(DateTime.DATETIME_MED)
```

Comment:
I guess the `formatDateFromISO` helper is still better here since it also considers the local language format.